route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManagerV5.cs

```diff
@@ -287,13 +292,17 @@ public AddressBookContact[] GetDepotsFromAddressBook(out ResultResponse resultRe
         {
             contactParams.PrepareForSerialization();
 
-            return GetJsonObjectFromAPIAsync<StatusResponse>(contactParams,
+            var result = GetJsonObjectFromAPIAsync<StatusResponse>(contactParams,
                 R4MEInfrastructureSettingsV5.ContactsAddMultiple,
                 HttpMethodType.Post,
                 null,
                 false,
                 false,
                 mandatoryNullableFields);
+
+            return Task.FromResult(
```

Comment:
What is the purpose of this change? I would recommend to avoid unnecessary memory traffic here. You can just return result or revert changes.

Reply:
See the comment above for similar question.